tests/apollo/util/bft.py

```diff
@@ -928,10 +924,12 @@ async def wait_for_state_transfer_to_stop(
                                 # Exit condition
                                 if n >= expected_seq_num:
                                     action.add_success_fields(n=n, expected_seq_num=expected_seq_num)
-                                    return
+                                    return None, True
 
                                 await trio.sleep(0.5)
-
+                
+        return await self.wait_for(retrieve_sequence_number, 30, .5)
```

Comment:
I'm not sure this one fits the pattern. The function only returns None, True which in combination with removing the tuple altogether probably doesn't work. Maybe we should just keep the old code or refactor it more completely to somehow fit in the wait_for paradigm.

Reply:
The function has been restored to it's previous state.